gui/src/app/Project/ProjectQueryLoading.ts

```diff
@@ -37,6 +38,7 @@ export const fromQueryParams = (searchParams: URLSearchParams) => {
     project: searchParams.get(QueryParamKeys.Project),
     stan: searchParams.get(QueryParamKeys.StanFile),
     data: searchParams.get(QueryParamKeys.DataFile),
+    'analysis.py': searchParams.get(QueryParamKeys.AnalysisPyFile),
```

Comment:
Yeah, this is an important choice (not just internal) because it's they key for the query string. I personally think analysis.py=... is the most intuitive here, even if it has a ".". Just preparing for analysis.r, data.py, data.r, and others (maybe samples.csv or initial_values.csv, etc)

Reply:
I think for a query string it can be a bit long form, I suggest “python_analysis”

Reply:
If we go that route, then we're going to continue having to think of special query names, when we already have a well-defined export to project. I still don't see the downside of using those exported file names in the query string. Then we only need one convention for these things, rather than unnecessary mappings.

Reply:
I personally find it confusing to see a “.” character in that part of a URL — I honestly didn’t think it would even be legal the first time I saw it

We already have mappings for all the other keys, which are not exactly their file name

Reply:
May I suggest analysis_py, analysis_r, etc? Then we're not using "." and the mapping is straightforward.

Reply:
Fine by me - @jsoules?

Reply:
Oh yeah, sounds great here.